src/help-bot/help-bot.knowledge.ts

```
@@ -431,6 +546,117 @@ function parseWalletCount(rawCount: string): number | null {
+  const normalizedQuestion = normalizeText(question);
+  const hasConsolidationContext = matchesAny(
+    normalizedQuestion,
+    CONSOLIDATION_CONTEXT_PATTERNS
+  );
+  return (
+    matchesAny(normalizedQuestion, WALLET_CONTEXT_PATTERNS) &&
+    matchesAny(normalizedQuestion, MULTI_WALLET_REMOVAL_ACTION_PATTERNS) &&
+    !hasDisqualifyingWalletContext(
+      normalizedQuestion,
+      hasConsolidationContext
+    )
```

ESLint

[error] Replace `⏎······normalizedQuestion,⏎······hasConsolidationContext⏎····` with `normalizedQuestion,·hasConsolidationContext`

(prettier/prettier)